cities/cities.json

```diff
@@ -175,6 +175,10 @@
         "id": "oldenburg",
         "label": "Oldenburg (Oldb)"
     },
+    "osnabrueck": {
```

Comment:
I appreciate your contribution. Please note that [we support German Umlauts as mentioned in the contribution guide](https://github.com/wo-ist-markt/wo-ist-markt.github.io/blob/master/CONTRIBUTING.md#market-data-contributions). Please rename _osnabrueck.json_ to _osnabrück.json_ and adapt the change in _cities.json_ as follows:

``` json
"osnabrück": {
    "id": "osnabrück",
    "label": "Osnabrück"
},
```

You are welcome to update this pull request. You can update your commit using `git commit --amend` and then update the branch on GitHub via `git push --force market-data-osnabrueck`. Let me know if you need help in any case.